tests/integration/local/start_api/test_start_api.py

```diff
@@ -382,14 +383,14 @@ def test_valid_v2_lambda_integer_response(self):
 
     @pytest.mark.flaky(reruns=3)
     @pytest.mark.timeout(timeout=600, method="thread")
-    def test_invalid_v2_lambda_response(self):
+    def test_v2_lambda_response_skip_unexpected_fields(self):
```

Comment:
I believe this change is not your changes, you got them when you merged from develop branch.

I prefer to split these commits in a separate PR, so we can merge them directly. It will be easy for us to review your changes only.